### ISSUE TITLE
A referred user should go through the join a campaign flow

### ISSUE THREAD

**Issue by shawnlauzon** [Volentix/venue#133]
**Describe the bug**
When I sign in as a referred user, I am not prompted to join a campaign. When I signin as a new user without a referral code, I go to the Join a Campaign screen

**To Reproduce**
1. User A invites User B
2. User B clicks on referral link, creates a new user, verifies email, and signs in
3. The dashboard is displayed
4. Click on Leaderboard: User B is not displayed

**Expected behavior**
After I signin for the first time (after verifying the email), I should be sent to the join campaign screen

**Screenshots**
n/a

**Additional context**
The URL of the join campaign screen is https://venue-dev.volentix.io/onboarding/bitcointalk

**Comment by Lyncis** [comment]
@shawnlauzon 
Did you use the referral link to register for user B in the same browser that user A was logged in?
If yes, did you log out as user A?

I repeated the steps:
1. Logged in as wolverine
2. Copied the referral link
3. Opened new tab, inserted the link and registered a new user.
4. Verified email
5. In the same browser opened a new tab, logged in as a new user.
6. It logged me in as wolverine (user A) and I guess it happened because I used the same browser and didn't log out from user A.
Therefore, I see the dashboard.

When I log out as user A and only then log in as user B, I get redirected to the join campaign page.

![image](https://user-images.githubusercontent.com/8875863/44117426-3bcb1370-9fe1-11e8-9da4-96c33cb68fa9.png)
